brouter-routing-app/src/main/java/btools/routingapp/BRouterActivity.java

```diff
@@ -76,6 +87,24 @@ public void onCreate( Bundle savedInstanceState )
     // Create a bright wake lock
     mWakeLock = mPowerManager.newWakeLock( PowerManager.SCREEN_BRIGHT_WAKE_LOCK, getClass().getName() );
 
+    // CHANGE: consider start with Intent to import a profile
+    Intent intent = getIntent();
+    String action = intent.getAction();
+    String type = intent.getType();
+    System.out.println("Brouter START xxxxxx BrouterActivity");
+    if (Intent.ACTION_VIEW.equals(action) ) {
+      System.out.println("Brouter Intent.ACTION_VIEW detected");
+      import_profile(intent);
+      return;
```

Comment:
It doens't run any of the other code in the BRouterActivity. Perhaps it would make sense to move it to an own Activity?

Reply:
Sure, create an own activity is possible...
I found this entry point, and it was running.
But in any case, you have to test in the same way (if (Intent...) to start the import_profile...